scripts/crypt-gpg-pinentry

```diff
@@ -15,10 +15,16 @@ if ($path[0] === '@') {
     // in a local PEAR tree that's not in the system PHP include path.
 }
 
-// Workaround for composer installs (#19914)
-$composerDir = $path . DIRECTORY_SEPARATOR . '..' . DIRECTORY_SEPARATOR . 'Console_CommandLine';
-if (is_dir($composerDir)) {
-    $path .= PATH_SEPARATOR . $composerDir;
+// Workaround for composer installs (#19914).  Path used if installed via composer using PEAR as a repository.
+$composerDir_pear = $path . DIRECTORY_SEPARATOR . '..' . DIRECTORY_SEPARATOR . 'Console_CommandLine';
+if (is_dir($composerDir_pear)) {
+    $path .= PATH_SEPARATOR . $composerDir_pear;
+}
+
+//Path if installed via composer without supplying a repository
```

Comment:
Add space after `//`

This is the path used if installed from packagist or using the default config for custom vcs or composer repositories.